x/emissions/keeper/keeper.go

```diff
@@ -1746,6 +1747,8 @@ func (k *Keeper) DripTopicFeeRevenue(ctx context.Context, topicId TopicId, block
 		}
 		newTopicFeeRevenue = val.SdkIntTrim()
 	}
+
+	ctx.Logger().Debug(fmt.Sprintf("Dripping topic fee revenue: block %d, topicId %d, oldRevenue %v, newRevenue %v", ctx.BlockHeight(), topicId, topicFeeRevenue, newTopicFeeRevenue))
```

Comment:
I am not aware of anywhere else where we are doing logger().Debug()

I'm not against doing it, seeing as we can always filter logs, but we should probably agree as a team that it is ok to leave debug logging statements in the codebase because I think up until now we had not

Reply:
Good point.
My view: I don't see why we shouldn't follow logging best practices lol.
Especially when other modules are extremely verbose at DEBUG level.